### PR TITLE
Remove a wait in tests

### LINKED ISSUE #1146: Test failing: Add/remove multiple files and upload. Expected <input> not to exist in the DOM
Happened here https://github.com/ChainSafe/files-ui/runs/2837287321?check_suite_focus=true

```
      AssertionError: Timed out retrying after 4000ms: Expected <input> not to exist in the DOM, but it was continuously found.
 |       at Context.eval (http://localhost:3000/__cypress/tests?p=cypress/tests/file-management-spec.ts:194:50)
```
image

### PR DESCRIPTION
**Pull request by Tbaut**
closes https://github.com/ChainSafe/files-ui/issues/1146

### CODE REVIEW

**Review by FSM1 — APPROVED** [review]

File: packages/files-ui/cypress/tests/file-management-spec.ts [hunk 1]
```diff
@@ -38,10 +40,15 @@ describe("File management", () => {
...
-      homePage.startUploadButton().should("not.be.disabled").click()
+      homePage.fileListRemoveButton().should("have.length", 2)
+      homePage.startUploadButton()
+        .should("not.be.disabled")
+        // this pipe is needed to prevent https://github.com/ChainSafe/files-ui/issues/1146
+        // as described https://www.cypress.io/blog/2019/01/22/when-can-the-test-click/
+        .pipe(click)
+        .should(($el: JQuery<HTMLElement>) => {
+          expect($el).to.not.be.visible
+        })
```
Comment (asnaith) [inline packages/files-ui/cypress/tests/file-management-spec.ts]: This is great @Tbaut 🙏

This is added to the `it` where we test the upload function directly but we should also replace the `wait` on on the convenience function in the page object with the same (`homePage.ts` / `uploadFile()`) 🙂

Comment (Tbaut) [reply packages/files-ui/cypress/tests/file-management-spec.ts]: ah, didn't realize we duplicated that. I made sure it's only used in one place.

File: packages/files-ui/cypress/tests/file-management-spec.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { binPage } from "../support/page-objects/binPage"
-import { homePage } from "../support/page-objects/homePage"
+import { click, homePage } from "../support/page-objects/homePage"
```
Comment (RyRy79261) [inline packages/files-ui/cypress/tests/file-management-spec.ts]: Codacy mentions that click is unused
```suggestion
import { homePage } from "../support/page-objects/homePage"
```

**Review by RyRy79261 — APPROVED** [review]

File: packages/files-ui/cypress/tests/file-management-spec.ts (hunk 2 above)
Comment (Tbaut) [reply packages/files-ui/cypress/tests/file-management-spec.ts]: Ah forgot to push that, thanks

**Pull request merged**